FND/Utilities/Analytics.rdf

```diff
@@ -397,6 +399,20 @@
 				<owl:allValuesFrom rdf:resource="&fibo-fnd-dt-fd;RecurrenceInterval"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;hasBaseDate"/>
```

Comment:
should have an explanatoryNote that only one of baseDate and basePeriod are expected to be used

Reply:
The FCT changed the names of these properties and added two additional properties to be clearer, but we need to review how they are applied.  hasBaseDate is now hasAnchorDate, which is the start for a series, to be clear about what that means.  We added hasFixedComparativeDate (e.g., the end of the last recession), hasRelativeComparativeDate (e.g. prior reported figure), and changed hasBasePeriod to hasRelativeComparativePeriod, again for clarity, but we need further analysis to determine whether they belong on scoped measure or something else.